frontend-react/src/components/FileHandlers/FileHandler.test.tsx

```diff
@@ -274,40 +261,47 @@ describe("FileHandler", () => {
 
                 renderWithFullAppContext(<FileHandler />);
 
-                fireEvent.change(
+                userEvent.upload(
                     screen.getByTestId("file-input-input"),
-                    fileChangeEvent
+                    fakeFile
                 );
-
                 await screen.findByTestId("file-input-preview-image");
 
-                fireEvent.click(screen.getByText("Validate"));
+                // jsdom seems currently unable to properly handle required file input
```

Comment:
Ugh, makes sense but unfortunate.  As an aside, I think it'd be worthwhile to disable native validations in favor of our own form validation logic anyway -- especially for more complex validators -- but thoughts for the future.